app/assets/stylesheets/application.css.scss

```diff
@@ -44,7 +50,22 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 }
 
 #footer {
-  font-size: 10px;
+  height: 20px;
```

Comment:
Properties should be ordered background-color, bottom, height, left, line-height, position, vertical-align, width